packages/pymupdf/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: pymupdf
+  version: 1.25.4
+  top-level:
+    - pymupdf
+    - fitz
+source:
+  url: https://files.pythonhosted.org/packages/25/56/d7de0325125621a3d095eb43ce35f2e036cd4c0489ff5e8cae816f1cd8b9/pymupdf-1.25.4.tar.gz
+  sha256: 5f189466b68901055a9ddc77dc1c91cba081a60964f0caa6ff5b9b87001a0194
+about:
+  home: https://pymupdf.readthedocs.io
+  PyPI: https://pypi.org/project/pymupdf
+  summary:
+    A high performance Python library for data extraction, analysis, conversion
+    & manipulation of PDF (and other) documents.
+  license: Dual Licensed - GNU AFFERO GPL 3.0 or Artifex Commercial License
+
+build:
+  exports: whole_archive
+  script: |
+    export PYMUPDF_SETUP_FLAVOUR=pb
+    export PYMUPDF_SETUP_MUPDF_TESSERACT=0
+    export PYMUPDF_SETUP_MUPDF_BUILD="git:--recursive --depth 1 --shallow-submodules --branch master https://github.com/ArtifexSoftware/mupdf.git"
+    export HAVE_LIBCRYPTO=no
+    export OS=pyodide
```

Comment:
Could you please add your name as a recipe maintainer here? Such as:

https://github.com/pyodide/pyodide/blob/d5be5797fb171bbe1b18fd17ccc2d993970a559d/packages/numpy/meta.yaml#L34-L36

There is no formal role for recipe maintainers, but since we have no knowledge of PyMuPDF, we may ask for your help when someone asks about this package.

Reply:
I started using this library a few days ago, not sure if I qualify as a maintainer 😅

Reply:
Well since you added the recipe you can count as the maintainer for the recipe. Recipe maintainers does not mean that you are a maintainer of the upstream package.